en/flight_modes/altitude_fw.md

```diff
@@ -0,0 +1,42 @@
+# Altitude Flight Mode (Fixed Wing)
+
+[<img src="../../assets/site/difficulty_2.svg" title="Difficulty (Easy)" width="30px" />](../getting_started/flight_modes.md#key_difficulty)&nbsp;[<img src="../../assets/site/remote_control.svg" title="Manual/Remote control required" width="30px" />](../getting_started/flight_modes.md#key_manual)&nbsp;
+
+The *Altitude* flight mode makes it easier for users to control vehicle altitude, and in particular to reach and maintain a fixed altitude. The mode does not use GPS, and hence will not attempt to hold the vehicle course against wind.
+
+The climb/descent rate is controlled via the pitch/elevator stick. Once centered the autopilot latches onto the current altitude and will maintain it during yaw/roll, and at any airspeed. The throttle input controls airspeed. 
+
+When all remote control inputs are centered (no roll, pitch, yaw, and ~50% throttle) the aircraft will return to straight, level flight (subject to wind) and keep its current altitude.
+
+The diagram below shows the mode behaviour visually (for a [mode 2 transmitter](../getting_started/rc_transmitter_receiver.md#transmitters-for-aircraft)).
+
+![Altitude Control FW](../../images/flight_modes/altitude_control_mode_fw.png)
+
+## Technical Summary
+
+
+> **Note**
+>  * This mode is *assisted*. The autopilot makes it easier for users to control the vehicle. 
+>  * Manual input is required (RC controller, or gamepad/thumbsticks through MAVLink).
+>  * The altitude is normally measured using a barometer, which may become inaccurate in extreme weather conditions. Vehicles that include a LIDAR/range sensor will be able to control altitude with greater reliability and accuracy. 
+
+
+## Parameters
+
+The mode is affected by the following parameters:
+
+Parameter | Description

```

Comment:
Are there any other relevant parameters for FW altitude mode?

Reply:
I think I might merge this and ask for additional review/information from @RomanBapst as a post process. Whatever gets merged now might not be "complete" but is no worse than our current content.